input/errors.go

```diff
@@ -0,0 +1,30 @@
+package input
+
+import (
+	"github.com/spacemonkeygo/errors"
+)
+
+var Error *errors.ErrorClass = errors.NewClass("InputError") // grouping, do not instantiate
+
+/*
+	Indicates that the input failed to obtain data that correctly
+	matches the hash specifying the input.  This may mean there have
+	been data integrity issues in the storage or transport systems involved.
+*/
+var InputHashMismatchError *errors.ErrorClass = Error.NewClass("InputHashMismatchError")
+
+/*
+	Indicates that the target filesystem (the one given to `Apply`) had some error.
+*/
+var TargetFilesystemUnavailableError *errors.ErrorClass = Error.NewClass("TargetFilesystemUnavailableError")
```

Comment:
We have an identically-named be string-indistinguishable error in the output package.

I merged this before realizing I hadn't left this comment, and I don't care strongly enough to hold this up.
Take some or no action as desired.